src/test/java/io/managed/services/test/rhoas/KafkaRhoasAclTests.java

```diff
@@ -0,0 +1,367 @@
+        LOGGER.info("verify that we aren't logged-in");
+        assertThrows(CliGenericException.class, () -> cli.listKafka());
+
+        LOGGER.info("login the CLI");
+        CLIUtils.login(vertx, cli, Environment.PRIMARY_USERNAME, Environment.PRIMARY_PASSWORD).get();
+
+        LOGGER.info("verify that we are logged-in");
+        cli.listKafka();
+    }
+
+    @SneakyThrows
+    public void createServiceAccount() {
+
+        LOGGER.info("create a service account");
+        serviceAccountSecret = CLIUtils.createServiceAccount(cli, SERVICE_ACCOUNT_NAME);
+
+        LOGGER.info("get the service account");
+        var sa = CLIUtils.getServiceAccountByName(cli, SERVICE_ACCOUNT_NAME);
+        LOGGER.debug(sa);
+
+        assertTrue(sa.isPresent());
+        assertEquals(sa.get().getName(), SERVICE_ACCOUNT_NAME);
+        assertEquals(sa.get().getClientId(), serviceAccountSecret.getClientID());
+
+        serviceAccount = sa.get();
+        serviceAccount.setSecret(serviceAccountSecret.getClientSecret());
+    }
+
+    @SneakyThrows
+    public void createKafkaInstance() {
+
+        LOGGER.info("create kafka instance with name {}", KAFKA_INSTANCE_NAME);
+        var k = cli.createKafka(KAFKA_INSTANCE_NAME);
+        LOGGER.debug(k);
+
+        LOGGER.info("wait for kafka instance: {}", k.getId());
+        kafka = CLIUtils.waitUntilKafkaIsReady(cli, k.getId());
+        LOGGER.debug(kafka);
+    }
+
+    @SneakyThrows
+
+    @Test(priority = 1)
+    public void testCreateAclForUser() {
+        LOGGER.info("Create ACL for secondary user");
+        cli.createAcl(DIFF_USER, AclOperation.ALL, AclPermissionType.ALLOW, TOPIC_NAME);
+        var aclList = cli.listACLs(DIFF_USER);
+        Optional<AclBinding> aclFiltered = CLIUtils.searchAcl(aclList, DIFF_USER, AclOperation.ALL, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
+        assertTrue(aclFiltered.isPresent());
+    }
+
+    @SneakyThrows
+    @Test(dependsOnMethods = "testCreateAclForUser", enabled = true)
+    public void testDeleteAclForUser() {
+        LOGGER.info("Create ACL for secondary user");
+        cli.deleteAcl(DIFF_USER, AclOperation.ALL, AclPermissionType.ALLOW);
+        var aclList = cli.listACLs(DIFF_USER);
+        Optional<AclBinding> aclFiltered = CLIUtils.searchAcl(aclList, DIFF_USER, AclOperation.ALL, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
+        assertTrue(aclFiltered.isEmpty());
+    }
+
+    @SneakyThrows
+    @Test(priority = 2)
+    public void testCreateAclForServiceAccount() {
+        LOGGER.info("Create ACL for service account");
+        cli.createAcl(serviceAccount, AclOperation.ALL, AclPermissionType.ALLOW, TOPIC_NAME);
+        var aclList = cli.listACLs(serviceAccount);
+        Optional<AclBinding> aclFiltered = CLIUtils.searchAcl(aclList, serviceAccount, AclOperation.ALL, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
+        assertTrue(aclFiltered.isPresent());
+    }
```

Comment:
the following two methods 
```
    @Test(priority = 1)
    public void testCreateAclForUser() {
        LOGGER.info("Create ACL for secondary user");
        cli.createAcl(DIFF_USER, AclOperation.ALL, AclPermissionType.ALLOW, TOPIC_NAME);
        var aclList = cli.listACLs(DIFF_USER);
        Optional<AclBinding> aclFiltered = CLIUtils.searchAcl(aclList, DIFF_USER, AclOperation.ALL, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
        assertTrue(aclFiltered.isPresent());
    }

    @SneakyThrows
    @Test(priority = 2)
    public void testCreateAclForServiceAccount() {
        LOGGER.info("Create ACL for service account");
        cli.createAcl(serviceAccount, AclOperation.ALL, AclPermissionType.ALLOW, TOPIC_NAME);
        var aclList = cli.listACLs(serviceAccount);
        Optional<AclBinding> aclFiltered = CLIUtils.searchAcl(aclList, serviceAccount, AclOperation.ALL, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
        assertTrue(aclFiltered.isPresent());
    }
```

are basically the same, the only difference is user vs service account which can be resolved by on of comments in CLI class. 

Same goes for basically all test down. So i think we should resolve this for this test and follow it accordingly. There is a simple solution for this using Data provider (while also making proritization of test unnecessary) 

so instead test could be written like this: 


```
@Test(dataProvider = "aclEntityTopicPairs")
public void testCreateAllowAllOnTopicAclForEntity(ACLEntityType entityType, topicName) {
    ....
```

with data provider: 
```
    @DataProvider(name = "aclEntityTopicPairs")
    public Object[][] aclEntityTopicPairsDataProvider() {
        return new Object[][]{
                {ACLEntityType.USER, "topic-user"},
                {ACLEntityType.SERVICE_ACCOUNT, "topic-service-acc"}};
    }
```